conf/scylla.yaml

```diff
@@ -71,6 +71,14 @@ commitlog_sync_period_in_ms: 10000
 # is reasonable.
 commitlog_segment_size_in_mb: 32
 
+# The size of the individual schema commitlog file segments.
+#
+# The default size is 128, which is 4 times larger than the default
+# size of the data commitlog. It's because the segment size puts
+# a limit on the mutation size that can be written at once, and some
+# schema mutation writes are much larger than average.
+schema_commitlog_segment_size_in_mb: 128
```

Comment:
Let's leave that at 32 for this series. The series and patch subject talk about adding a configuration item, but changing the actual size is much larger.

Or you can add a third patch that changes the value and change the series topic to be about that. It's much more important than the configurability.

Reply:
I've added a third patch that changes the value from 32 to 128 and have renamed the series topic to be about that.